Missing script

**I'm submitting a bug report**

* **Operating System:**
Windows [10]

* **Node Version:**
7.8.0

* **NPM Version:**
4.4.4

**Current behavior:**

```
C:\Projetos\ux
λ  npm run test-watch
npm ERR! missing script: test-watch

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\alber\AppData\Roaming\npm-cache\_logs\2017-04-17T14_43_23_181Z-debug.log
```

* **What is the expected behavior?**

Run the **test-watch** npm script

--- a/README.md
+++ b/README.md
@@ -27,5 +27,5 @@ npm run test
 ## Developing
 
 ```shell
-npm run test-watch
+npm run develop
 ```

